components/TextFields/examples/TextFieldUnderlineExample.swift

```diff
@@ -151,15 +151,8 @@ final class TextFieldUnderlineSwiftExample: UIViewController {
     scrollView.addSubview(message)
     let messageController = MDCTextInputControllerUnderline(textInput: message)
     message.textView?.delegate = self
-    #if swift(>=3.2)
-      message.text = """
-      This is where you could put a multi-line message like an email.
-
-      It can even handle new lines.
-      """
-    #else
-      message.text = "This is where you could put a multi-line message like an email. It can even handle new lines./n"
-    #endif
+    message.text = "This is where you could put a multi-line message like an email." + "\n\n" +
```

Comment:
Why did this switch to not using `"""` syntax?

Reply:
I got tripped up about how it should be formatted and changed it to this way because I felt more sure of it, but I can change it back if you'd like

Reply:
I'd prefer to keep the original source and only make the changes required for 4.2 migration.

Reply:
Fair enough 👍

Reply:
Let's leave it. The Presubmits take too long. Sorry!